README.md

```diff
@@ -160,7 +160,7 @@ In case you want to use client certificates, set the `client_cn` accordingly.
 
 ### Dashing Configuration
 
-Edit `config/icinga2.json` and adjust the settings for the Icinga 2 API credentials.
+Move `config/icinga2.json.example` to `config/icinga2.json` and adjust the settings for the Icinga 2 API credentials.
```

Comment:
Mh, I prefer to have a defined configuration in place, although I understand the idea. This should be discussed in a separate issue imho, to move the configuration into a sample.